nx_parallel/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .utils.decorators import _configure_if_nx_active
 from .utils import *
 from .algorithms import *
+from .algorithms import components
```

Comment:
This shouldn't be needed if we import inside `algorithms/__init__.py`, right?  I think this will provide `nx_parallel.components` instead of `nx_parallel.algorithms.components`.